src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -493,17 +624,58 @@ def setup(self):
         self.encoder = FlaxBertEncoder(self.config, dtype=self.dtype)
         self.pooler = FlaxBertPooler(self.config, dtype=self.dtype)
 
-    def __call__(self, input_ids, attention_mask, token_type_ids, position_ids, deterministic: bool = True):
+    def __call__(
+        self,
+        input_ids,
+        attention_mask,
+        token_type_ids,
+        position_ids,
+        deterministic: bool = True,
+        output_attentions: bool = False,
+        output_hidden_states: bool = False,
+        return_dict: bool = True,
+    ):
         hidden_states = self.embeddings(
             input_ids, token_type_ids, position_ids, attention_mask, deterministic=deterministic
         )
-        hidden_states = self.encoder(hidden_states, attention_mask, deterministic=deterministic)
+        outputs = self.encoder(
+            hidden_states,
+            attention_mask,
+            deterministic=deterministic,
+            output_attentions=output_attentions,
+            output_hidden_states=output_hidden_states,
+        )
+        hidden_states = outputs[0]
+        pooled = self.pooler(hidden_states) if self.add_pooling_layer else None
+
+        outputs = (
+            hidden_states,
+            pooled,
+        ) + outputs[1:]
```

Comment:
```suggestion
        outputs = (hidden_states, pooled) + outputs[1:]
```
This fits in one line ;-)